imports/plugins/core/orders/server/no-meteor/mutations/createRefund.js

```diff
@@ -0,0 +1,111 @@
+import SimpleSchema from "simpl-schema";
+import accounting from "accounting-js";
+import Logger from "@reactioncommerce/logger";
+import ReactionError from "@reactioncommerce/reaction-error";
+import { getPaymentMethodConfigByName } from "/imports/plugins/core/payments/server/no-meteor/registration";
+import sendOrderEmail from "../util/sendOrderEmail";
+
+const inputSchema = new SimpleSchema({
+  amount: Number,
```

Comment:
Does it make sense to set a minimum of 0 for amount?

Reply:
Maybe not `0`, but maybe `0.01`, if it's possible to do that. I can't think of a circumstance where there is s `0` refund.